marimo/_cli/export/commands.py

```diff
@@ -335,7 +341,75 @@ def export_callback(file_path: MarimoPath) -> ExportResult:
     return watch_and_export(MarimoPath(name), output, watch, export_callback)
 
 
+@click.command(
+    help="""Export a notebook as a WASM-powered standalone HTML file.
+
+Example:
+
+  \b
+  * marimo export html-wasm notebook.py -o notebook.wasm.html
+
+The exported HTML file will run the notebook using WebAssembly, making it
+completely self-contained and executable in the browser.
+
+In order for this file to be able to run, it must be served over HTTP,
+and cannot be opened directly from the file system (e.g. file://).
+"""
+)
+@click.option(
+    "-o",
+    "--output",
+    type=str,
+    required=True,
+    help="""Output directory to save the HTML to.""",
+)
+@click.option(
+    "--mode",
+    type=click.Choice(["edit", "run"]),
```

Comment:
In this context `run` isn't self explanatory. How about `readonly`?

Reply:
i was thinking it aligns with `marimo edit` and `marimo run`